wqu_app/app.py

```diff
@@ -15,7 +16,7 @@ def main():
     else:
         ip_address = retrieve_local_ip_adress()
 
-    return render_template('index.html', message=greet(ip_address))
+    return render_template('index.html', message=greet(ip_address)[0], graph = greet(ip_address)[1])
```

Comment:
I am going to be picky and point out there shouldn't be a space around the `=` for keyword args. https://www.python.org/dev/peps/pep-0008/#other-recommendations

Also, we are calling `greet` twice. Let's just make it one call. There are several ways to do this, call the function before the `return` line or key-word unpacking: `**dict(zip(('message', 'graph'), greet(ip_address))`